script.js

```diff
@@ -2,11 +2,39 @@ const button = document.getElementById('add');
 const list = document.getElementById('list');
 const textArea = document.getElementById('new-task');
 
-function generateId() {
-  const id = new Date();
-  return id.getTime();
+generateId = () => {
+  return Date.now();
 }
```

Comment:
Pójdźmy jeszcze o krok dalej. Da się to zwinąć do jednej linijki? Skoro jedyne co siedzi w ciele funkcji to return?

Reply:
done